packages/react-jsx-highcharts-gantt/src/components/Pathfinder/index.js

```diff
@@ -0,0 +1,4 @@
+import Pathfinder from './Pathfinder';
+export default Pathfinder;
+
+// TODO: Does Pathfinder need sub components for the types of the properties: startMarker, endMarker and marker?
```

Comment:
> The component will have no children, so you can just return null from render

Actually ignore what I said above, probably does make sense to make sub components out of these.

I that case, the above Pathfinder component should  `return this.props.children` instead

Reply:
You mean that I should only ignore what you said about "the component will have no children"?

Reply:
Yeah, sorry that could have been clearer!

Reply:
No worries, just asked to be sure... trying to implement your other comments now. Thanks for your great help so far!

Reply:
No problem, great to see it coming along!